CHANGELOG.md

```
@@ -1,6 +1,15 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+### [Version 7.8.1](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/7.8.1) (August 7, 2026)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)